docs/assets/scripts/hedwig-main.umd.js

```diff
@@ -6567,8 +6568,11 @@
       this.attachShadow({
         mode: 'open'
       });
-      this.shadowRoot.appendChild(svg);
-      this.renderGraph(this.parseData(data), svg);
+
```

Comment:
Is this necessary to add the same code in this file?

Reply:
auto-generated

Reply:
yes .. so I don't think it needs to be committed from local!